plugin/grepper.vim

```diff
@@ -1079,10 +1079,10 @@ endif
 " Commands {{{1
 command! -nargs=* -complete=customlist,grepper#complete Grepper call <sid>parse_flags(<q-args>)
 
-for tool in g:grepper.tools
-  let utool = toupper(tool[0]) . tool[1:]
-  execute 'command! -nargs=+ -complete=file Grepper'. utool
-        \ 'Grepper -noprompt -tool' tool '-query <args>'
+for s:tool in g:grepper.tools
```

Comment:
Uh.. guess I have to check for that in _all_ my plugins now...

(Note to self: there is no block scope in VimL.)